packages/frontend/scss/login/_login.scss

```diff
@@ -201,25 +201,29 @@ div.delta-form-group {
   }
 }
 
-progress.delta-progress-bar {
+div.delta-progress-bar {
   width: 100%;
+  background-color: var(--progressBarBg);
+  height: 8px;
+  margin-block: 21pt;
+  --progressBarBorderRadius: 5px;
+  border-radius: var(--progressBarBorderRadius);
```

Comment:
Is there really a point to this variable? Why not just `99999px` or `50%`?

Again, feel free to ignore.

Reply:
used in two places, so using var makes it easy to iterate on in the inspector.

Reply:
Oh. I missed the other place. Apologies.